### PR TITLE
Add `noOverviewImages` docs

### PR DESCRIPTION
**Pull request by fiskus**
- [x] Documentation

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1204042699719463

### CODE REVIEW

File: docs/Catalog/VisualizationDashboards.md [hunk 1]
```diff
@@ -97,6 +97,22 @@ use the following extended syntax:
...
+`./config.js`. See the Quilt Catalog
+[README.md](https://github.com/quiltdata/quilt/tree/master/catalog#configuration)
+for details.
+
+On the Bucket tab, we show images from the current directory in the current
+nesting level.
+
+On the Packages tab we show images similarly to the Bucket tab but inside the
+package.
+
```
Comment (robnewman) [inline docs/Catalog/VisualizationDashboards.md:115]:
```suggestion
## Images

If your Amazon S3 bucket contains images, by default the Quilt Catalog displays a preview of those images _before_ any `quilt_summarize.json`-referenced files.

In the **Overview** tab, the Catalog parses the entire Amazon S3 bucket contents and displays thumbnail image previews in a paginated grid (25 per page by default) of all [supported image types](../Catalog/Preview.md#binary-and-special-file-format-previews).

> To hide this block, specify the `noOverviewImages` flag in your [Bucket configuration file](../Catalog/Preferences.md).

In the **Bucket** tab, the Catalog displays thumbnail image previews in a similarly paginated grid but _only from the current directory viewed_.

In the **Packages** tab, when a specific package has been opened the Catalog displays thumbnail image previews in a similarly paginated grid but _only those image files in the selected package_.

```

**Review by robnewman — CHANGES_REQUESTED** [review]
I don't see the `noOverviewImages` in the Catalog/Preferences.md file? I thought configuration was only in `.quilt/catalog/config.yaml`? Are you referencing `.quilt/catalog/config.js`?

File: docs/Catalog/VisualizationDashboards.md (hunk 1 above)
Comment (fiskus) [reply docs/Catalog/VisualizationDashboards.md:115]: `noOverviewImages` is not in the Bucket config file (config.yyml per bucket), it's in the confi.js (main config file per stack). I know it's not the right place, I just described what we have already

Comment (fiskus) [reply docs/Catalog/VisualizationDashboards.md:115]: This is the config file http://open.quiltdata.com/config.js

Comment (robnewman) [reply docs/Catalog/VisualizationDashboards.md:115]: Okay - cleaned up. Thanks for clarifying.

**Review by robnewman — APPROVED** [review]

**Pull request merged**